### ISSUE TITLE
Generate a simple table indicating software versions

### ISSUE THREAD

**Issue by dleehr** [Duke-GCB/lando#123]
During meeting on September 5, 2018, client requested to have a simple table of used software and versions in the output project. 

The rationale being that this would be an easy way to tell what was used to generate the data, apart from walking through the packed CWL, reading the HTML methods document, or visiting the website to review the description

The request from the client was a simple two-column CSV file such as:

```
"Software","Version"
"FastQC","0.11.4"
"GATK","3.8"
...
```

I agree that having such information be easily discoverable is a good idea. I think this should be a part of reworking our reports documents.

Also related: https://github.com/Duke-GCB/bespin-cwl/issues/33

One thing that's not clear to me - does this relate to #32 ?

**Comment by johnbradley** [comment]
bespin-api contains classes that read the software version data out of the packed workflow and build the methods document. This is done when importing a new workflow version.

https://github.com/Duke-GCB/bespin-api/blob/c18746d9a9cc657733f4cbc71348946a783cc2d9/data/importers.py#L101-L124

https://github.com/Duke-GCB/bespin-api/blob/c18746d9a9cc657733f4cbc71348946a783cc2d9/data/importers.py#L248-L250

**Comment by dleehr** [comment]
Good point. We discussed this in last monday's meeting. Since we have structured data (the CWL workflow) with software versions as well as the results+methods documents, we need to clarify what else would be accomplished by such an addition, that wouldn't be satisfied by some refinements to the reporting
